pages/guide.js

```diff
@@ -119,6 +126,53 @@ function Home(props) {
           ))}
         </div>
       </section>
+      <section>
+        <h2 className="h5">Components</h2>
+
+        <h3 className="h6">IconCard</h3>
+        <div
+          style={{
+            display: "grid",
+            gap: "1.25rem",
+            gridTemplateColumns: "repeat(auto-fill, minmax(300px, 1fr))",
+          }}
+        >
+
+              <IconCard
+                title="Decentralized"
+                icon="decentralized"
+                copy={
+                  "Not controlled by a single website or company, Mastodon is a network of completely independent service providers forming a global, cohesive social media platform. "
+                }
+              />
```

Comment:
I was thinking about if the `FormattedMessage` component should be passed as props (as it is currently) or used as react `{children}`

```
              <IconCard
                title={<FormattedMessage id="..." defaultMessage="..." />}
                icon="decentralized"
                copy={<FormattedMessage id="..." defaultMessage="..."  />}
              />
```
vs
```
              <IconCard icon="decentralized">
                <h3><FormattedMessage id="..." defaultMessage="..."  /></h3>
                <p><FormattedMessage id="..." defaultMessage="..."  /></p>
              </IconCard>
```

Though I suppose the latter would mean you could put anything/any style in the card, so prob not desirable.

Reply:
right and now that the styling is done in tailwind, i think we almost _have_ to use renderprops so the styling isn't redefined each time it's used